.circleci/config.yml

```diff
@@ -197,7 +197,7 @@ workflows:
               matrix:
                  parameters:
                     os: [ linux ]
-                    python_version: [ "3.8", "3.9", "3.10", "3.11", "3.12" ]
+                    python_version: [ "3.10" ]
```

Comment:
Is there any future compatibility issue created by pinning to a single version?

Reply:
This PR was meant for testing only.  I do not plan to merge it.  I reduced the number of Python versions so that only a small number of CI processes are launched.